### PR TITLE
feat: Add RE7 config getter

### PR DESCRIPTION
**Pull request by ltyu**
### Description
Adds RE7 config getter

### Backward compatibility
Yes

### Testing
Manual

### CODE REVIEW

**Review by xeno097 — APPROVED** [review]

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumZircuitRe7LRTWarpConfig.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const ISM_CONFIG = ethers.constants.AddressZero; // Default ISM
+
+export const getEthereumZircuitRe7LRTWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const ethereum: HypTokenRouterConfig = {
+    ...routerConfig.ethereum,
+    owner: Re7Safes.ethereum,
+    proxyAdmin: {
```
Comment (nambrot) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumZircuitRe7LRTWarpConfig.ts:28]: Is this override required? I thought the top level owner is sufficient?

File: typescript/infra/src/config/warp.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export const tokens: ChainMap<Record<string, Address>> = {
     USDT: '0xdac17f958d2ee523a2206206994597c13d831ec7',
     WBTC: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599',
     weETHs: '0x917cee801a67f933f2e6b33fc0cd1ed2d5909d88',
+    Re7LRT: '0x84631c0d0081FDe56DeB72F6DE77abBbF6A9f93a',
```
[the remark this answers is not in the document]
Comment (nambrot) [reply typescript/infra/src/config/warp.ts:15]: I'm not sure I understand this comment?

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumZircuitRe7LRTWarpConfig.ts (hunk 1 above)
Comment (ltyu) [reply typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumZircuitRe7LRTWarpConfig.ts:28]: yes, you are right

**Review by nambrot — APPROVED** [review]

File: typescript/infra/config/environments/mainnet3/warp/warpIds.ts [hunk 3]
```diff
@@ -35,4 +35,5 @@ export enum WarpRouteIds {
   ArbitrumEthereumMantleModePolygonScrollZeroNetworkUSDT = 'USDT/arbitrum-ethereum-mantle-mode-polygon-scroll-zeronetwork',
   ArbitrumBaseEthereumLiskOptimismPolygonZeroNetworkUSDC = 'USDC/arbitrum-base-ethereum-lisk-optimism-polygon-zeronetwork',
   ArbitrumBaseBlastBscEthereumGnosisLiskMantleModeOptimismPolygonScrollZeroNetworkZoraMainnet = 'ETH/arbitrum-base-blast-bsc-ethereum-gnosis-lisk-mantle-mode-optimism-polygon-scroll-zeronetwork-zoramainnet',
+  EthereumZircuitRe7LRT = 'Re7LRT/ethereum-zircuit',
```
Comment (paulbalaji) [inline typescript/infra/config/environments/mainnet3/warp/warpIds.ts:38]: nit: could you group this with the other `Ethereum` routes pls 🙏

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**